addons/airway/functions/fnc_treatmentAdvanced_Accuvac.sqf

```diff
@@ -25,8 +25,21 @@ params ["_medic", "_patient", "_bodyPart", "_classname", "", "_usedItem"];
 if !(_patient getVariable [QGVAR(occluded), false]) exitWith {
     private _output = LLSTRING(Accuvac_NA);
     [_output, 1.5, _medic] call ACEFUNC(common,displayTextStructured);
+
+    if (_usedItem isEqualTo "kat_suction") then {
+        _medic addItem "kat_suction";
+    };
+};
+
+if (_usedItem isEqualTo "kat_suction") then {
+    if(GVAR(Suction_reuse)) then {
+        _medic addItem "kat_suction";
+    };
 };
 
-[_patient, _classname] call ACEFUNC(medical_treatment,addToTriageCard);
+[_patient, "Airway Suction"] call ACEFUNC(medical_treatment,addToTriageCard);
```

Comment:
The class name looks good, so delete that line.

Reply:
By the way, did you know that if you use a class name instead of English, it will be written to triagecard as "ACCUVAC" and "SUCTION" in any language?
Even in other medical procedures, most of the information is written in English or by class name.
Or do need to go to the trouble of using German and Japanese or some languages for only airway suctions?

I chose "Airway Suction" to show in the log that ACCUVAC and SUCTION are doing the same thing (Airway Suction).
Did you want the log to show ACCUVAC and SUCTION separately?

as classname in not English langage config:
![20240125064752_1](https://github.com/KAT-Advanced-Medical/KAM/assets/50947830/4c23c5a5-18f0-471a-971b-e19955a01d28)

as "airway suction" in not English langage config:
![20240125065409_1](https://github.com/KAT-Advanced-Medical/KAM/assets/50947830/c2a1e8f0-c8e1-49b3-b3c6-6832f8f3688e)

Reply:
Other examples of using English instead of class names https://github.com/KAT-Advanced-Medical/KAM/blob/78669c517b6915b7887f5a9766f7d2d1e420e9d2/addons/pharma/functions/fnc_treatmentAdvanced_DialysisLocal.sqf#L27

Reply:
https://github.com/KAT-Advanced-Medical/KAM/pull/453/commits/8cc4c0475d19507dc266394bd7e7bc788acb60de
This change looks ok.